pkg/controller/controller.go

```diff
@@ -88,8 +93,21 @@ func NewController(client kubernetes.Interface,
 	return c
 }
 
+//NewEmptyController is an empty constructor
+func NewEmptyController() *Controller {
```

Comment:
Do we really need this helper?  I see you're using it in the test but I can't quite figure out why the test can't just do "c := &Controller{}" on it's own?